src/v3/src/components/QRCode/QRCode.tsx

```diff
@@ -14,23 +14,26 @@ import { Box } from '@mui/material';
 import { withTheme } from '@okta/odyssey-react-theme';
 import { h } from 'preact';
 import { QRCodeElement, UISchemaElementComponent } from 'src/types';
+import { getTranslation } from 'src/util';
```

Comment:
Couldnt run locally because of this, had to change to `'../../util'`

Reply:
bacon also caught this issue, going to fix it now